### PR TITLE
Surface command output when running request.command

### PR DESCRIPTION
**Pull request by niclim**
## 🍗 Description
_What does this PR do? Anything folks should know?_

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

File: projects/optic/src/commands/capture/actions/captureRequests.ts [hunk 1]
```diff
@@ -195,14 +195,18 @@ async function runRequestsCommand(
...
+        resolve();
       } else {
         reject();
       }
     });
   });
 
   reqCmd.stdout.on('data', (data) => {
-    logger.debug(data.toString());
+    logger.info(data.toString());
```
Comment (niclim) [inline projects/optic/src/commands/capture/actions/captureRequests.ts:209]: I think it's correct here to long data as it comes out in `info` - where server only logs on `debug`. Reasoning here is that the command _may_ also be tests that you want to see the test output in. 

We might also need to do some formatting / handle interactions with the spinner here, but I think that can come later

**Comment by notnmeyer** [comment]
example output,

```
➜ npx ts-node ../optic/projects/optic/src/index.ts capture openapi.yml
⠋ Sending requests to server{"message":"Hello, world!"}
[{"id":1,"name":"John"},{"id":2,"name":"Bob"},{"id":3,"name":"Alice"}]
{"id":4,"name":"Hank"}
✔ Successfully captured requests

3 unmatched interactions
New endpoints are only added in interactive mode.
Run with `--update interactive` to add new endpoints
Hint: optic capture openapi.yml --update interactive
```

i kind of ignored how ugly the output can be when the output was behind debug, but if we're displaying this by default then we should add a newline after "Sending requests to server"

**Comment by niclim** [comment]
Hmmm i agree, it's a little more complicated with how the spinner works (the spinner assumes it's the last thing to print to stdout), let me see what i can do here, i saw some workarounds from the ora github issues page

**Comment by notnmeyer** [comment]
depending on how annoying that workaround is, we could probably just insert an `echo;` before the contents of `requests.run.command`

**Comment by niclim** [comment]
@notnmeyer I updated this so that the spinner is always flushed to the bottom until it is completed (i.e. everything inside of the create requests should be better spaced so we're logging in the right areas)

```
➜  my-openapi-specs git:(main) ✗ npx ts-node ../optic/projects/optic/src/index.ts capture abe.yml 
asdasdasdasda
 a
 a 
 a 
 a 
 a 
 a 


sd

✔ Successfully captured requests
✔ GET /books
  ✓ 200 response

3 unmatched interactions
```

**Review by notnmeyer — APPROVED** [review]
yep, works for me

**Pull request merged**